cognite/extractorutils/configtools.py

```diff
@@ -249,6 +249,84 @@ def __repr__(self) -> str:
         return self._expression
 
 
+class FileSizeConfig(yaml.YAMLObject):
+    def __init__(self, expression):
+        self._bytes, self._expression = FileSizeConfig._parse_expression(expression)
```

Comment:
Maybe we should `expression` as `extension` instead.
```suggestion
        self._bytes, self._extension = FileSizeConfig._parse_expression(expression)
```

Reply:
It's the whole expression, unaltered, and not just the unit :) It's stored to be returned back on `str(...)` for example.